src/corebluetooth.cs

```diff
@@ -665,6 +673,9 @@ interface CBUUID : NSCopying {
 		[Field ("CBUUIDCharacteristicAggregateFormatString")]
 		NSString CharacteristicAggregateFormatString { get; }
 
+		[Field ("CBUUIDCharacteristicValidRangeString")]
+		NSString CharacteristicValidRangeString { get; }
```

Comment:
was there a `StrongDictionary` using the other existing fields ? if so then it will need to be updated too